python3: can't open file 'fictest.py': [Errno 2] No such file or directory
